lua/bufferline/render.lua

```diff
@@ -646,12 +646,6 @@ function render.restore_buffers(bufnames)
   for _,name in ipairs(bufnames) do
     table_insert(state.buffers, bufadd(name))
   end
-
-  -- Update after a delay - sometimes buffer names aren't updated
-  local timer = vim.loop.new_timer()
-  timer:start(500, 1, vim.schedule_wrap(function()
-    render.update()
-  end))
```

Comment:
`vim.schedule(function() render.update(true) end)` is done on `SessionLoadPost` now, so I have removed it from this function to reduce duplication.